Formula/f/fakesteak.rb

```diff
@@ -0,0 +1,17 @@
+class Fakesteak < Formula
+  desc "ASCII Matrix-like steak demo"
+  homepage "https://github.com/domsson/fakesteak"
+  url "https://github.com/domsson/fakesteak/archive/refs/tags/v0.2.4.tar.gz"
+  sha256 "0f183a2727e84d2e128e3192d4cff3e180393c9c39b598fa1c4bfe8c70a4eb1a"
+  license "CC0-1.0"
+
+  def install
+    system "make"
+    bin.install "bin/fakesteak"
```

Comment:
```suggestion
    system "make", "PREFIX=#{prefix}", "install"
```
I think this should work.

Reply:
yeah, it works 👍